R/data_interface.R

```diff
@@ -350,16 +350,14 @@ spark_data_write_generic <- function(df, path, fileMethod, mode = NULL, csvOptio
 #' @export
 spark_read_table <- function(sc,
                              name,
-                             path,
                              options = list(),
                              repartition = 0,
                              memory = TRUE,
                              overwrite = TRUE) {
 
   if (overwrite) spark_remove_table_if_exists(sc, name)
 
-  operation <- if (spark_version(sc) < "2.0.0") "load" else "table"
-  df <- spark_data_read_generic(sc, path, operation, options)
+  df <- spark_data_read_generic(sc, name, "table", options)
```

Comment:
@edgararuiz Well, we don't have fallback behavior since the previous behavior was to save this with `path=name` which will save the data on the current directory with the given `name` as `path`, but you know what, we can throw an `unsupported` error when running in `master` mode. Let me add that.